src/Uno.UI/UI/Xaml/ElementStub.netstd.cs

```diff
@@ -22,11 +22,16 @@ private FrameworkElement MaterializeContent()
 
                 if (currentPosition != -1)
                 {
-                    parentElement.RemoveChild(this);
+					// Create the instance first so that x:Bind constructs can be picked up by the
+					// Unload event of ElementStub. Not doing so does not fills up the generated variables
```

Comment:
```suggestion
					// Unload event of ElementStub. Not doing so fills up the generated variables
```

Reply:
Fixing this in a different PR as this one is time sensitive